test/verify/check-metrics

```diff
@@ -1026,6 +1026,52 @@ BEGIN {{
 
         m.execute("rm /tmp/hogged")
 
+        # Test Podman containers
+        container_name = "pod-mem-hog"
+        m.execute(f"podman run --rm -d --name {container_name} quay.io/libpod/busybox /bin/sh -c 'cat /dev/zero | head -c 300m | tail | sleep infinity'")
+
+        container_sha = m.execute(f"podman inspect --format '{{{{.Id}}}}' {container_name}").strip()
+        shortid = container_sha[:12]
+
+        # On some test images the container takes a while to show up
+        with b.wait_timeout(300):
+            b.wait_text("table[aria-label='Top 5 memory services'] tbody tr:nth-of-type(1) td[data-label='Service']", f"pod {shortid}")
+
+        # It takes one re-render for the name lookup
+        with b.wait_timeout(30):
+            b.wait_text("table[aria-label='Top 5 memory services'] tbody tr:nth-of-type(1) td[data-label='Service']", f"pod {container_name}")
+
+        m.execute(f"podman stop -t 0 {container_name}")
```

Comment:
Does `-t 0` exist on RHEL 8? ISTR that it doesn't, in console.dot I had to add a few hacks for that. Well, tests will tell!

Reply:
We already had `-t 0` in the CPU tests so this should be fine.